src/Uno.UI/UI/Xaml/Controls/ComboBox/ComboBox.cs

```diff
@@ -142,6 +142,7 @@ protected override void OnLoaded()
 		{
 			base.OnLoaded();
 
+			UpdateCommonStates();
```

Comment:
Can you test if it fixes the bug if you put this in `OnApplyTemplate()` instead?

If so it should cover some additional edge cases (eg, ComboBox is both Collapsed and disabled, and becomes Visible after it is already loaded)

Reply:
Yes, I will test it tomorrow morning. I thought of OnApplyTemplate first but saw the UpdateDropDownState in the OnLoaded and continue to test at this level only after. But yes I will let you know the outcome after my test.

Reply:
Works well in `OnApplyTemplate()` after all the tests, I correctly updated the code for it.
Thanks again for all the great feedback @davidjohnoliver :) !